.gitignore

```diff
@@ -2,3 +2,5 @@
 .sass-cache
 _site
 .DS_Store
+vendor
```

Comment:
These directories are created by bundle when running locally. Instructions added in https://github.com/mimblewimble/site/pull/39.